src/03_modules.py

```diff
@@ -5,27 +5,43 @@
 level operating system functionality.
 """
 
+import os
 import sys
 # See docs for the sys module: https://docs.python.org/3.7/library/sys.html
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
+for items in sys.argv:
+    print(items)
 
 # Print out the OS platform you're using:
 # YOUR CODE HERE
+# print("OS Platform you're using:", sys.platform)
+
+if sys.platform.startswith('win32'):
+    print("Looks like you're using Windows")
+elif sys.platform.startswith('linux'):
+    print("Looks like you're using Linux")
+elif sys.platform.startswith('cygwin'):
+    print("Looks like you're using Windows/Cygwin")
+elif sys.platform.startswith('darwin'):
+    print("Looks like you're using MacOS")
 
```

Comment:
Nice custom printing here.  Just make sure you know all the possible inputs, or create an else statement to catch any OS types that might not be covered.